content/data.md

```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
 
 ## Packages
 
+### Maintainer Gender 2017
+
+In [Pinto, Wiese & Dias (2018). How Do Scientists Develop Scientific Software? An External Replication](http://gustavopinto.org/lost+found/saner-rene2018.pdf), the authors surveyed CRAN maintainers that self-identified as a 
+scientist. The survey was conducted starting in late November 2017 and ending in early January 2018. The respondents were contacted 
+by email and asked to complete an online questionnaire, with a response rate of 24% (a total of 1,553 responses). 
+
+<div class="commentbox"> In 2017, 10.9% of package maintainers who identified as scientists were women. </div>
+
+- 44.6% of respondents identify between 30 and 40 years of age at the time of the survey.
+
+- Overwhelmingly, 80.0% of respondents identified as either having completed or are in progress of completing a doctorate degree. 
```

Comment:
having completed or are in progress of completing a doctorate degree -> having a completed or in-progress doctorate degree.

Reply:
Incorporated in 56e8927